# Defined in udev rules, change according to your system. Typically /dev/ttyUSB0
serial_port: /dev/ttyUSB0

# Acceptable baud rates : 2400, 4800, 9600, 19200, 38400, 57600, 115200, 230400, 460800, 921600
serial_baud: 115200

# Acceptable data rates in Hz: 0.2Hz, 0.5Hz, 1Hz, 2Hz, 5Hz, 10Hz, 20Hz, 50Hz, 100Hz, 200Hz
# Baud rate must be able to handle the data rate
pub_rate: 20

# Frame id to publish data in
frame_id: imu_link

# Data publication form, true for East North Up or false for North East Down <- false is default setting
tf_ned_to_enu: true

# If publishing ENU, do we want to publish in the frame labeled on the device? Default is false.
# If tf_ned_to_enu = true and frame_based_enu = false orientation is reported by: x->y y->x z->-z to rotate the quaternion
# Proper method is to rotate the quaternion by multiplication
# If tf_ned_to_enu = true and frame_based_enu = true we rotate the quaternion to the frame matched label by multiplication
frame_based_enu: false

# Topic names configurations
imu_topic: /wit/imu_data
mag_topic: /wit/mag_data
temperature_topic: /wit/temperature_data
pressure_topic: /wit/pressure_data

# Make sure all covariances below are of type xx.xx , i.e. double so that the rpc is parsed correctly

# Linear Acceleration Covariances not produced by the sensor
linear_accel_covariance: [0.000001,  0.0,    0.0,
                          0.0,   0.000001,   0.0,
                          0.0,   0.0,    0.000001]

# Angular Velocity Covariances not produced by the sensor
angular_vel_covariance: [0.000001,  0.0,  0.0,
                         0.0,  0.000001,  0.0,
                         0.0,  0.0,  0.000001]

# Orientation covariance overwritten in driver, this is included just as an extra
orientation_covariance: [0.000001,  0.0,   0.0,
                         0.0,   0.000001,  0.0,
                         0.0,   0.0,   0.000001]

# Magnetic Covariances not produced by the sensor
magnetic_covariance: [0.000001,  0.0,  0.0,
                      0.0,  0.000001,  0.0,
                      0.0,  0.0,  0.000001]